toolchain/sem_ir/name_scope.h

```diff
@@ -72,40 +153,25 @@ struct NameScope : Printable<NameScope> {
   // vector so that these can pack densely. If this ends up both cold and memory
   // intensive, we can also switch the lookup to a set of indices into the
   // vector rather than a map from `NameId` to index.
-  llvm::SmallVector<Entry> names;
-  Map<NameId, int> name_map;
+  llvm::SmallVector<Entry> names_;
+  Map<NameId, int> name_map_;
```

Comment:
```suggestion
  Map<NameId, EntryId> name_map_;
```

Thinking about this further, since you're adding the `EntryId` type, maybe we should use it in the `Map` too? I thought about this reading `LookupOrAdd`, I think it'd reduce a bit of the `EntryId` wrapping there.

Reply:
Done.